pkg/agent/openflow/packetin.go

```diff
@@ -108,7 +114,7 @@ func (c *client) subscribeFeaturePacketIn(featurePacketIn *featureStartPacketIn)
 	return nil
 }
 
-func (c *client) parsePacketIn(packetInQueue workqueue.Interface, packetHandlerReason uint8) {
+func (c *client) parsePacketIn(packetInQueue workqueue.RateLimitingInterface, packetHandlerReason uint8) {
```

Comment:
can we call `packetInQueue.Forget(obj)` after `packetInQueue.Done(obj)` even though it is a no-op (and add a comment to mention it is a no-op for the `BucketRateLimiter` implementation